demo/app-root.ts

```diff
@@ -1,19 +1,87 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+import { html, css, LitElement, customElement, query } from 'lit-element';
+import '../src/iaux-underlined-tab-bar';
+import type { UnderlinedTabBar } from '../src/iaux-underlined-tab-bar';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @query('iaux-underlined-tab-bar') tabBar!: UnderlinedTabBar;
+
+  private itemClicked(e: CustomEvent) {
+    console.log(e);
+    console.log(e.detail.index);
+  }
+
+  private doSomething() {
```

Comment:
would it be more expressive to name this method something like `setEntries()` or `changeOptions()`?